README.md

```diff
@@ -94,3 +148,36 @@ jobs:
       # Use the standard extra args and ignore spread tests
       trivy-extra-args: '--skip-dirs "tests/spread/**"'
 ```
+
+## Python test runner
+
+The Python test runner workflow uses GitHub workflows and `uv` to run Python tests in
+several forms. It:
+
+#. Runs fast tests across multiple platforms and Python versions.
+#. Runs all tests on Ubuntu with the oldest supported python version and uv resolution
+set to `lowest`.
+#. Runs slow tests across their own set of platforms and Python versions.
+#. Uploads test coverage for tests as artefacts.
```

Comment:
This list is not properly rendered in markdown. See https://github.com/lengau/starflow/tree/work/CRAFT-3602/test-workflows. 

```suggestion
- Runs fast tests across multiple platforms and Python versions.
- Runs all tests on Ubuntu with the oldest supported python version and uv resolution
set to `lowest`.
- Runs slow tests across their own set of platforms and Python versions.
- Uploads test coverage for tests as artefacts.
```